packages/suite/src/components/suite/banners/SuiteBanners/SuiteBanners.tsx

```diff
@@ -20,6 +20,7 @@ import styled from 'styled-components';
 
 const Container = styled.div`
     background: ${({ theme }) => theme.backgroundSurfaceElevation0};
+    border-bottom: solid 1px ${({ theme }) => theme.borderOnElevation0};
```

Comment:
I would rather dynamically add a border above the page, to accommodate the coinjoin status bars too :) Although that also does not look ideal but then the Banner change is out of scope anyways 😓 

![image](https://github.com/trezor/trezor-suite/assets/45338719/d03a029f-2273-487c-b617-cccc2de294a1)

Reply:
It needs to be conditional if it's above the page, and I'm not sure if we can assure it.

Reply:
I think it's minor and we can create separate issue for it, WDYT?